game-cards/SGgame.js

```diff
@@ -382,13 +382,20 @@ exports.commands = {
 			type = "Fire";
 		} else if (starters.indexOf(target) >= 14) {
 			type = "Water";
-		}
+		} else if (starters.indexOF(target) = 15) {
+			type = "Electric";
+		} else if (starters.indexOF(target) = 16) {
+			type = "Normal";
 		if (type === "Grass") {
```

Comment:
This chain of `if else` should be removed and merged in with the other set of `if else` statements. ex:
```javascript
if (starters.indexOf(target) <= 6) {
    type = "Grass";
    typeColor = "green";
} else if (starters.indexOf(target) <= 13) { // ect...
```